modules/openapi-generator/src/main/resources/python/model_utils.mustache

```diff
@@ -799,18 +799,19 @@ def remove_uncoercible(required_types_classes, current_item, spec_property_namin
             results_classes.append(required_type_class)
     return results_classes
 
-def get_discriminated_classes(cls):
+def get_discriminated_classes(cls, seen_discriminator, possible_classes):
```

Comment:
Can we remove possible classes as an input? Why are we routing it in here? Isn't possible_classes only in the context of the current schema that we are looking at?